lib/katello/plugin.rb

```diff
@@ -203,6 +203,16 @@
                    :onlyif => proc { |proxy| proxy.has_feature?(SmartProxy::PULP_FEATURE) || proxy.has_feature?(SmartProxy::PULP_NODE_FEATURE) }
   end
 
+  register_facet Katello::Host::ContentFacet, :content_facet do
```

Comment:
@lzap, one thing that I could not test was apipie generation. There was an issue with apipie that the documentation was generated before the plugin could register its facets, so Katello properties were omitted from Host documentation